src/main/java/com/gmail/mrphpfan/mccombatlevel/PlayerCombatLevelChangeEvent.java

```diff
@@ -0,0 +1,77 @@
+package com.gmail.mrphpfan.mccombatlevel;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+
+/**
+ * Called when a players combat level changes
+ * 
+ * @author Zettelkasten
+ *
+ */
+public class PlayerCombatLevelChangeEvent extends PlayerEvent implements Cancellable {
+
+	private final static HandlerList handlers = new HandlerList();
+	
+	private final int oldLevel;
+	private int newLevel;
+	
+	private boolean cancel;
+
+	public PlayerCombatLevelChangeEvent(Player player, int oldLevel, int newLevel) {
+		super(player);
+
+		this.oldLevel = oldLevel;
+		this.newLevel = newLevel;
+		
+		this.cancel = false;
```

Comment:
This statement. Field variables compared to local variables automatically get a default value, if none value is specified for that variable. So in case of a boolean, it's "false". Therefore there is no need for this statement.